daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/DFDLEscapeScheme.scala

```diff
@@ -36,8 +36,11 @@ import org.apache.daffodil.lib.schema.annotation.props.PropertyLookupResult
 import org.apache.daffodil.lib.schema.annotation.props.NotFound
 import org.apache.daffodil.lib.schema.annotation.props.Found
```

Comment:
If you expand the diff to look at all the imports, you can see scalafmt did not sort the imports.  I wonder if something is missing from scalafmt's configuration.  BTW, when you try to change the configuration to sort the imports, I would suggest basing your changes on the v4 commit so you can run scalafmt and see from "git diff" more quickly whether scalafmt sorted the imports or not without seeing all the v4 changes obscuring the imports.

Reply:
This was a little confusing. There is a separate option that also needs to be included:

    rewrite.rules = [Imports]

and that only sorted imports there were on the same line. If I enable group sorting, then it sorts "vertically". I've set this to create three separate groups:

    rewrite.imports.groups = [
        ["scala\\..*", "java\\..*", "javax\\..*"],
        ["org\\.apache\\.daffodil\\..*"],
    ]

Standard libraries first, then daffodil, and followed by everything else.